src/main/java/org/jscep/message/PkiMessageDecoder.java

```diff
@@ -63,6 +65,11 @@ public PkiMessageDecoder(final X509Certificate signer,
             final PkcsPkiEnvelopeDecoder decoder) {
         this.decoder = decoder;
         this.signer = signer;
+        this.checkSignatureTime = true;
+    }
+
+    public void NoSignatureTimeCheck() {
```

Comment:
Can you rename this method as ignoreSigningTime please?

Reply:
hopefully done